### PR TITLE
Add custom aggregation units API

### LINKED ISSUE #219: Support get statistics per aggregation unit
News: https://developers.line.biz/en/news/2021/03/10/messaging-api-update-march-2021/#202103-02
Doc: https://developers.line.biz/en/reference/partner-docs/#get-statistics-per-unit

### PR DESCRIPTION
**Pull request by yskkin**
closes #219

### CODE REVIEW

File: lib/line/bot/client.rb [hunk 1]
```diff
@@ -999,6 +999,45 @@ def update_audience_authority_level(authority_level)
...
+        channel_token_required
+
+        endpoint_path = "/bot/message/aggregation/info"
+        get(endpoint, endpoint_path, credentials)
+      end
+
+      # Get the name list of units used this month for statistics aggregation.
+      #
+      # @param limit [Integer] Maximum number of aggregation units per request. Maximum: 100, Default: 100.
+      # @param start [String] Value of the continuation token found in the next property of the JSON object returned in the response.
```
Comment (zenizh) [inline lib/line/bot/client.rb:1030]: [nice to have]
It is better to express the property name as a code.

```suggestion
      # @param start [String] Value of the continuation token found in the `next` property of the JSON object returned in the response.
```

File: lib/line/bot/client.rb [hunk 2]
```diff
@@ -999,6 +999,45 @@ def update_audience_authority_level(authority_level)
...
+      #
+      # @return [Net::HTTPResponse]
+      def get_unit_names(limit: nil, start: nil)
```
Comment (zenizh) [inline lib/line/bot/client.rb:1033]: [nice to have]
This method name is very generic and could be the same as any other endpoint. A method name based on the endpoint would be preferred.

```suggestion
      def get_aggregation_list(limit: nil, start: nil)
```

File: lib/line/bot/client.rb [hunk 3]
```diff
@@ -999,6 +999,45 @@ def update_audience_authority_level(authority_level)
...
+        channel_token_required
+
+        params = {limit: limit, start: start}.compact
+        endpoint_path = "/bot/message/aggregation/list?" + + URI.encode_www_form(params)
```
Comment (zenizh) [inline lib/line/bot/client.rb:1037]: [must fix]
Unnecessary `+` is present. This is not an error, but should be removed.

```suggestion
        endpoint_path = "/bot/message/aggregation/list?" + URI.encode_www_form(params)
```

**Review by zenizh — CHANGES_REQUESTED** [review]
Thanks for the contribution! I've made a few comments to check out. cc: @kimoto

**Review by zenizh — APPROVED** [review]
Thank you for fixing about my suggestion. LGTM!

**Comment by kimoto** [comment]
Sorry, Too late response. I will confirming it.

**Review by kimoto — APPROVED** [review]

**Pull request merged**